distribution/lib/Standard/Microsoft/0.0.0-dev/src/SQLServer_Details.enso

```diff
@@ -0,0 +1,48 @@
+from Standard.Base import all
+import Standard.Base.Data.Numbers.Number_Parse_Error
+import Standard.Base.Errors.Illegal_State.Illegal_State
+
+import Standard.Database.Connection.Connection_Options.Connection_Options
+import Standard.Database.Connection.Credentials.Credentials
+
+import project.Internal.SQLServer_Connection.SQLServer_Connection
+
+type SQLServer_Details
+    ## Connect to a SQLServer database.
+
+       Arguments:
+       - host: The name of the server to connect to.
+       - port: The port to connect to.
+       - credentials: The credentials to use for the connection.
+       - database: The name of the database to connect to.
+    SQLServer host:Text port:Text credentials:Credentials database:Text="master"
+
+    ## PRIVATE
+       Build the Connection resource.
+
+       Arguments:
+       - options: Overrides for the connection properties.
+    connect : Connection_Options -> SQLServer_Connection
+    connect self options =
+        properties = options.merge self.jdbc_properties
+
+        ## Cannot use default argument values as gets in an infinite loop if you do.
```

Comment:
Is this comment still relevant? I'm not sure if I understand why an infinite loop would happen here.

Reply:
Not sure. It is copied pasted... Postgres and Snowflake both have the same comment. Not that that makes it correct :)

Reply:
Removed it from here